build.sbt

```diff
@@ -38,6 +38,27 @@ libraryDependencies ++= Seq(
   "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % Test
 )
 
+// Overidden transient dependencies for Vulnerability fixes
+libraryDependencies ++= Seq(
+  // Introduced through com.typesafe.play:play_2.13:2.9.0
+  // No newer version of play available yet.
+  "com.typesafe.akka" %% "akka-actor" % "2.8.1",
+  "com.typesafe.akka" %% "akka-actor-typed" % "2.8.1",
+  "com.typesafe.akka" %% "akka-protobuf-v3" % "2.8.1",
+  "com.typesafe.akka" %% "akka-serialization-jackson" % "2.8.1",
+  "com.typesafe.akka" %% "akka-slf4j" % "2.8.1",
+  "com.typesafe.akka" %% "akka-stream" % "2.8.1",
```

Comment:
Are we sure we want to do these? Note that [Play 2.9 stuck with Akka 2.6](https://www.playframework.com/documentation/3.0.x/Highlights29#Akka-HTTP-10.2) because of the new licensing, I think if we use Akka above that version we might be breaking licensing compliance. Perhaps best to jump to Play 3 and Pekko...

Reply:
Don't think we'd be in breach of the license as we're not using these packages directly, they're used by Play which is specifically permitted by the License

https://www.lightbend.com/akka/license

Will try bumping this PR to Play 3.0 now though, might not be much work.

Reply:
Disregard, we do infact use Akka directly 😅 we'll definitely need to wait for Play 3.0!